docs-site/config.toml

```diff
@@ -33,6 +33,7 @@ highlight_themes_css = [
   { theme = "OneHalfDark", filename = "syntax-theme-dark.css" },
   { theme = "OneHalfLight", filename = "syntax-theme-light.css" },
 ]
+extra_syntaxes_and_themes = ["syntaxes"]
```

Comment:
can you show before and after this change?

Reply:
Before
<img width="769" height="524" alt="Screenshot 2025-07-20 at 15 37 36" src="https://github.com/user-attachments/assets/88b0e968-5c53-4aea-95e0-4ef5bff519af" />

After
<img width="749" height="542" alt="Screenshot 2025-07-20 at 15 38 23" src="https://github.com/user-attachments/assets/5126e5f0-0e0b-4e49-9046-6ac09a3dd03d" />

Adding custom syntaxes to highlight is documented [here](https://www.getzola.org/documentation/content/syntax-highlighting/)

Reply:
ok and why we need `docs-site/syntaxes/Terraform.sublime-syntax` file? it not should be in zola?